README.md

```diff
@@ -127,6 +127,8 @@ The following sections provide a full list of configuration in- and output varia
 | cluster\_fqdn | The FQDN of the created cluster |
 | cluster\_name | The name of the created cluster |
 | cluster\_resource\_group | Resource group name that contains AKS managed cluster |
+| cluster\_node\_resource\_group | Resource group name that contains AKS VMs |
+| network\_resource\_group |  Resource group name that contains AKS network resources |
```

Comment:
Can you keep this list in alpha order please?

Reply:
@chrismellard done